### ISSUE TITLE
Automatic Page Breaks

### ISSUE THREAD

**Issue by calnexus** [delight-im/HTML-Sheets-of-Paper#19]
Is there a way to add automated page breaks like in MS word without relying on the manual .page CSS rule when the page height is exceeded?

**Comment by ocram** [comment]
Thanks!

The live demo, which loads `/index.html`, has some JavaScript added that aims to take care of this. Have you tried it or seen it?

**Comment by calnexus** [comment]
Yes I have ,it doesn't work. Maybe i have some other library blocking the code.I managed to fix it by using javascript timers
// window.addEventListener("DOMContentLoaded", function() {
//     applyPageBreaks();
// });

setTimeout(() => {
    applyPageBreaks();
}, 1000);
